core/src/main/java/com/redhat/lightblue/client/LightblueClientConfiguration.java

```diff
@@ -32,6 +32,7 @@ public static Compression parseCompression(String str) {
     private ReadPreference readPreference;
     private String writeConcern;
     private Integer maxQueryTimeMS;
+    private transient Execution execution;
```

Comment:
Maybe it is not. I just thought since it was composed from some of the other fields that there would be no need to ever serialize it. Not a big deal though, I can remove it if you would prefer?